simple_network_sim/data.py

```diff
@@ -0,0 +1,14 @@
+from io import TextIOWrapper
+
+import pandas as pd
+from data_pipeline_api.file_api import FileAPI
+
+
+class Datastore(FileAPI):
+    def read_table(self, data_product: str) -> pd.DataFrame:
```

Comment:
Just a note for everyone's benefit, the standard API currently has component in here as well, which doesn't have meaning for csv files so has been omitted here. Assuming the standard API remains stable we will have to add that argument to the code when we integrate with the standard API.